backend/database_scripts/basic_search.sql

```diff
@@ -0,0 +1,33 @@
+CREATE VIEW basic_search AS
+SELECT 
+	A.overall_status status, 
+	A.brief_title, 
+	A.official_title,
+	A.nct_id,
+	B.names condition_name,
+	C.description study_detailed_desc, 
+	D.names country_name,
+	E.names intervention_name, 
+	F.names keywords,
+	G.criteria eligibility_criteria,
+	H.names location_name,
+	I.description study_brief_desc
+FROM 
+	studies A
```

Comment:
Depending on how often we will need to review the SQL, it would be advantageous to use clearer aliasing within the tables. Column names have great aliasing.

Reply:
Can you please give me an example in case better aliasing needs to be used?

Reply:
Partial name aliasing  (preferred)
FROM
    studies  stds
LEFT JOIN 
    ctgov.all_conditions cnds ON .....
LEFT JOIN 
    ctgov.detailed_descriptions dtldesc  ......


Full name aliasing

FROM
    studies 
LEFT JOIN 
    ctgov.all_conditions conditions ON ......
LEFT JOIN
    ctgov.detailed_descriptions detailDescription